azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CURL.java

```diff
@@ -44,6 +48,14 @@ private static String getEncodedURL(@NonNull String url) {
         }
     }
 
+    public static void validateURL(String url) {
+        try {
+            new URL(url);
+        } catch (MalformedURLException e) {
+            throw new AADB2CConfigurationException("Invalid logoutSuccessUrl: " + url, e);
```

Comment:
Will this `validateURL` be used to check `logoutSuccessUrl` only? If not, the error message may be misleading.

Reply:
good catch, should update the exception message.